controller/space.go

```diff
@@ -53,6 +54,26 @@ func (c *SpaceController) Create(ctx *app.CreateSpaceContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
+		/*
```

Comment:
@hectorj2f The code is good at explaining what it does, but not very good at why...